Spine/Operation.swift

```diff
@@ -219,14 +219,17 @@ class SaveOperation: ConcurrentOperation {
 	/// Whether the resource is a new resource, or an existing resource.
 	private let isNewResource: Bool
 	
-	private let relationshipOperationQueue = NSOperationQueue()
+	private var relationshipOperationQueue: NSOperationQueue?
```

Comment:
It's not necessary, but `removeObserver` throws an exception if the observer wasn't added in the first place. Because of that I need to track whether the observer is actually registered to the queue (the queue is only necessary if the object is being modified and there are relationship attributes in the model). The easiest is to make sure the queue only exists when it's needed.